### DISCUSSION TITLE
could u add a new language “lzh”？

### DISCUSSION THREAD

**Discussion by nonameawa** [Zrips/CMI#7714] [Enhancement]
“lzh” is the classic chinese. I am a classic chinese translator. So i wanna to translation the CMI to lzh. I dont know how 2 use github well.

**Comment by CloudeLecaw** [comment]
if you had translated cmi and cmilib into lzh allready then feel free to share the translation with us.
if a translations is shared by the comunity only then there is a chance that it might be added to cmi.
